File: src/main/java/com/procurement/notice/model/tender/record/TenderDescription.java
package com.procurement.notice.model.tender.record;

public enum TenderDescription {

    PS("Preselection stage of contracting process"),
    PQ("Preselection stage of contracting process"),
    PN("Contracting process is planned"),
    PIN("Date of tender launch is determined");

    private final String text;

    TenderDescription(final String text) {
        this.text = text;
    }

    public String getText() {
        return this.text;
    }
}

File: src/main/kotlin/com/procurement/notice/model/tender/record/TenderDescription.kt
package com.procurement.notice.model.tender.record

enum class TenderDescription(val text: String) {

    PS("Preselection stage of contracting process"),
    PQ("Prequalification stage of contracting process"),
    EV("Evaluation stage of contracting process"),
    PN("Contracting process is planned"),
    PIN("Date of tender launch is determined")
}
